### DISCUSSION TITLE
older git versions

### DISCUSSION THREAD

**Discussion by neorasp** [community/community#23021] [New to GitHub]
<p>hello world<br>
i want to try using an older version of git to compile projects to work on winxp. where can i find ready builds?</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/neorasp">@neorasp</a> </p>
<p>The latest version of git for windows with support for XP was 2.10.0 which you can download <a href="https://github.com/git-for-windows/git/releases/tag/v2.10.0.windows.1" rel="nofollow noopener">here</a></p>

**Comment by neorasp** [comment]
<p>Thank you <a class="mention" href="/seveas">@seveas</a> , this was what i was looking for</p>
